### PR TITLE
Add Scaling IT docs

### PR DESCRIPTION
**Pull request by azexcy**
Changes proposed in this pull request:
- Add Scaling IT docs

### CODE REVIEW

File: docs/document/content/reference/test/scaling-integration-test/_index.cn.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+weight = 4
++++
+
+## 测试目的
+
+验证Scaling自身功能和依赖模块的正确性。
+
+## 测试环境
+
+环境准备方式分为 Native 和 Docker，不论哪种环境，本地都需要预先安装Docker，
```
Comment (sandynz) [inline docs/document/content/reference/test/scaling-integration-test/_index.cn.md:12]: The last '，' could be improved.

File: docs/document/content/reference/test/scaling-integration-test/_index.cn.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+- Native环境 用于本地调试，可以使用IDE的debug模式进行调试
```
Comment (sandynz) [inline docs/document/content/reference/test/scaling-integration-test/_index.cn.md:14]: Every english word should be seperated by blank, e.g. `Native_`, `_IDE_`, `_debug_`.
And also other places.

File: docs/document/content/reference/test/scaling-integration-test/_index.cn.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+- Docker环境  环境由 Maven 运行，适用于云编译环境和测试 ShardingSphere-Proxy 的场景，如：GitHub Action
+
+当前默认采用 Docker 环境，涉及到的 ShardingSphere-Proxy, Zookeeper, 数据库实例(MySQL,PostgreSQL), 都通过Docker自动启动。
```
Comment (sandynz) [inline docs/document/content/reference/test/scaling-integration-test/_index.cn.md:17]: ',' could be replaced to '，' or '、'

File: docs/document/content/reference/test/scaling-integration-test/_index.cn.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+
+主键用例：
+
+- TextPrimaryKeyScalingIT: 支持主键为文本类型的表迁移
+
+
+### 配置文件
+
+目录：`resources/env/`
+- /common: 存放scaling过程中用到的Dist SQL
```
Comment (sandynz) [inline docs/document/content/reference/test/scaling-integration-test/_index.cn.md:39]: `Dist SQL` could be `DistSQL`

File: docs/document/content/reference/test/scaling-integration-test/_index.cn.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+
+因此Native模式下不支持运行多个case，每次跑完需要自行清理Zookeeper中的信息，以及重启ShardingSphere-Proxy。
+
+启动方式如下: 找到需要测试的Case，比如 MySQLGeneralScalingIT, 在启动之前配置对应的 VM Option，新增如下配置
+```
+-Dit.cluster.env.type=native -Dit.env.mysql.version=${image-name}
+```
+> 其中的image-name需要是合法的docker image名称，比如：mysql:5.7, 多个的话用逗号隔开
+
+在IDE下使用Junit的方式启动即可
```
Comment (sandynz) [inline docs/document/content/reference/test/scaling-integration-test/_index.cn.md:59]: Could be ended with '。'

File: docs/document/content/reference/test/scaling-integration-test/_index.cn.md [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+**Docker 环境配置为 ShardingSphere-Proxy 提供了远程调试端口，默认是3308。**
+可以在 ShardingSphereProxyDockerContainer 中自行修改
+
+#### 运行单个用例
+
+和Native一样，只需要改一个参数。
+
+```
+-Dit.cluster.env.type=docker -Dit.env.mysql.version=${image-name}
+```
```
Comment (sandynz) [inline docs/document/content/reference/test/scaling-integration-test/_index.cn.md:81]: Could be merged to `Native 环境启动`

File: docs/document/content/reference/test/scaling-integration-test/_index.cn.md [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+
+#### 运行多个用例
+
+```shell
+./mvnw -nsu -B install -f shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-scaling/pom.xml -Dit.cluster.env.type=DOCKER -Dit.env.mysql.version=mysql:5.7,mysql:8.0 -Dit.env.postgresql.version=postgres:12-alpine
+```
```
Comment (sandynz) [inline docs/document/content/reference/test/scaling-integration-test/_index.cn.md:87]: Could be merged to `Docker 环境启动`, seprated to MySQL and PostgreSQL for better debugging, and also just keep one docker image for every database type.

File: docs/document/content/reference/test/scaling-integration-test/_index.cn.md [hunk 8]
```diff
@@ -0,0 +1,91 @@
++++
+title = "Scaling集成测试"
```
Comment (TeslaCN) [inline docs/document/content/reference/test/scaling-integration-test/_index.cn.md:2]: A space is required between Chinese and English character.

**Review by sandynz — CHANGES_REQUESTED** [review]
Thanks. There's some styles could be improved.

**Review by sandynz — APPROVED** [review]

**Pull request merged**